lib/disability_compensation/factories/api_provider_factory.rb

```diff
@@ -116,8 +122,19 @@ def ppiu_service_provider
     end
   end
 
+  def claims_service_provider
+    case api_provider
+    when API_PROVIDER[:evss]
+      EvssClaimsServiceProvider.new(nil, @options[:auth_headers])
+    when API_PROVIDER[:lighthouse]
+      LighthouseClaimsServiceProvider.new(@options[:icn])
+    else
+      raise NotImplementedError, 'No known Claims Service Api Provider type provided'
+    end
+  end
+
   def api_provider
-    @api_provider ||= if Flipper.enabled?(@feature_toggle, @current_user)
+    @api_provider ||= if @feature_toggle.present? && Flipper.enabled?(@feature_toggle, @current_user)
```

Comment:
we can remove `if @feature_toggle.present? &&` since `@feature_toggle` is now required in the constructor

Reply:
fixed